lib/tasks/queue.rake

```diff
@@ -40,4 +40,20 @@ namespace :queue do
 
     RequeueContent.new(scope).call
   end
+
+  desc "Preview of the message published onto rabbit MQ"
+  task :preview_last_message, [:document_type] => :environment do |_, args|
+    document_type = args[:document_type]
+    raise ValueError("expecting content_id") unless document_type.present?
+
+    scope = Edition
```

Comment:
I don't know if this is the correct way of working out what was last put onto the queue, since unpublished editions are also put onto the queue now. Perhaps a better way might be to use the last `Action` to determine the edition?

Reply:
I have renamed the message to "preview_recent_message" as discussed